### PR TITLE
add seperate tokenizer manager for fast fields

### PR DESCRIPTION
**Pull request by PSeitz**

### CODE REVIEW

File: src/schema/text_options.rs [hunk 1]
```diff
@@ -25,8 +25,12 @@ pub struct TextOptions {
 
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 #[serde(untagged)]
+/// Enum to control how the fast field setting of a text field.
 enum FastFieldOptions {
+    /// Flag to enable/disable
     IsEnabled(bool),
+    /// Enable with tokenizer. The tokenizer must be available on the fast field tokenizer manager.
+    /// `Index::tokenizer_fast_field`.
     EnabledWithTokenizer { with_tokenizer: TokenizerName },
```
Comment (fulmicoton) [inline src/schema/text_options.rs:34]: Do we call it normalizer or something like that?

Comment (PSeitz) [reply src/schema/text_options.rs:34]: I don't think we should restrict it to normalizer on tantivy side

File: src/core/index.rs [hunk 2]
```diff
@@ -409,6 +411,16 @@ impl Index {
         &self.tokenizers
     }
 
+    /// Setter for the fast field tokenizer manager.
+    pub fn set_fast_field_tokenizers(&mut self, tokenizers: TokenizerManager) {
+        self.fast_field_tokenizers = tokenizers;
+    }
+
+    /// Accessor for the fast field tokenizer manager.
+    pub fn tokenizer_fast_field(&self) -> &TokenizerManager {
```
Comment (fulmicoton) [inline src/core/index.rs]:
```suggestion
    pub fn fast_field_tokenizer(&self) -> &TokenizerManager {
```

.. or the other way around, but we should be more consistent

File: src/schema/text_options.rs (hunk 1 above)
Comment (fulmicoton) [reply src/schema/text_options.rs:34]: you are right

**Review by fulmicoton — APPROVED** [review]
approved but can you align naming one way or another before merging?

**Pull request merged**